.github/workflows/performance.yml

```diff
@@ -0,0 +1,56 @@
+# Merge the yml file to master branch for the cron job schedule to be effective.
```

Comment:
If there's no compelling reason to move this file also to public, I'd recommend leaving it on private only so that PRs can get immediate feedback.

I'm doubtful the nightly cron is worth the tradeoff, if that's the only reason.

Reply:
Cron job specified in this file guarantees the external dependencies of Fireperf is not breaking Fireperf iOS SDK, giving confidence on the healthiness of the product, and raise alert if breakage happens. Therefore it is needed for including this file for nightly build and test.